app/services/search/tag.rb

```diff
@@ -2,6 +2,7 @@ module Search
   class Tag
     INDEX_NAME = "tags_#{Rails.env}".freeze
     INDEX_ALIAS = "tags_#{Rails.env}_alias".freeze
+    MAPPING = YAML.load_file("config/elasticsearch_mappings/tags.yml").deep_symbolize_keys.freeze
```

Comment:
I wonder if it would be easier to keep this in JSON, since that's more in line with ES. 

```ruby
JSON.parse(File.read("config/elasticsearch_mappings/tags.json"), symbolize_names: true).freeze
```

I have no problem with YAML though.